certbot-apache/certbot_apache/parser.py

```diff
@@ -458,6 +468,24 @@ def find_dir(self, directive, arg=None, start=None, exclude=True):
 
         return ordered_matches
 
+    def get_all_args(self, match):
+        """Tries to fetch all arguments for a directive. See get_arg."""
+
+        if match[-1] != "/":
+            match = match+"/"
+        results = []
```

Comment:
Yeah, that's much more elegant. It will, however when run against a parent node (for example: `.../IfModule[1]/`) return the child directive names as well. I made sure to include a note about this in the pydoc.